### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by FranJRG**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
   <title>Welcome to Potionfy</title>
 </head>
 <body>
-  
+  <h1>Welcome to Potionfy! Enjoy your stay.<h1>
```
Comment (Rafmormor3) [inline index.html:10]: Could you remove the "Enjoy your stay." string and make the "Welcome to Potionfy!", string red?

Comment (Rafmormor3) [inline index.html]: Could you remove the "Enjoy your stay.", string and make the "Welcome the Potionfy!" string red?

Comment (FranJRG) [reply index.html]: Sure, can do!

Comment (Rafmormor3) [reply index.html]: Please, change the code as per my comments on the review.

**Review by Rafmormor3 — CHANGES_REQUESTED** [review]

**Review by Rafmormor3 — APPROVED** [review]

**Pull request merged**